lib/membrane/video_compositor/compound/base_size.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Membrane.VideoCompositor.Compound.BaseSize do
```

Comment:
1. Why `BaseSize`? I mean - what the `Base` refers to here?
2. Isn't it generic enough to define it at the compositor's top level?